### ISSUE TITLE
How long does it take to pretrain a large model?

### ISSUE THREAD

**Issue by vincentt1999** [shiyu-coder/Kronos#43]
thanks

**Comment by shiyu-coder** [comment]
Here is the training curve of Kronos-base for your reference. The sample size for each epoch is 1 million. The S1 and S2 losses in the figure represent the training losses of coarse and fine tokens, respectively. Although there was no significant change in training loss during the training process, the validation loss continued to decrease and stopped decreasing after around 50 epochs.

<img width="1409" height="912" alt="Image" src="https://github.com/user-attachments/assets/0de97cbd-fdd7-4e07-b529-a62fc54be645" />

**Comment by vincentt1999** [comment]
Have you considered continuing your training for a longer time? it seems could be better
